src/main.c

```diff
@@ -54,11 +100,28 @@ int main(int argc, const char* argv[]) {
 
   if (argc == 1) {
     repl(&vm);
-  } else if (argc == 2) {
+  } 
+  
+  else if (argc == 2) {
+    if (0 == strcmp(argv[1], "-h") || 0 == strcmp(argv[1], "--help")) {
```

Comment:
it makes more sense to check if the comparison is equal to 0 rather than if 0 is equal to the comparison